Makefile

```diff
@@ -0,0 +1,12 @@
+.venv:
```

Comment:
I like this approach!  (Makefiles are not dead in the post-C world apparently :) )